werkzeug/serving.py

```diff
@@ -43,6 +43,11 @@
 import signal
 
 try:
+    import termcolor
+except ImportError:
+    pass
```

Comment:
If termcolor couldn't be imported, then accessing it later in `if termcolor` will crash.